library/core/src/ptr/const_ptr.rs

```diff
@@ -1270,8 +1270,8 @@ impl<T: ?Sized> *const T {
     /// # fn foo(n: usize) {
     /// # use std::mem::align_of;
     /// # unsafe {
-    /// let x = [5u8, 6u8, 7u8, 8u8, 9u8];
-    /// let ptr = x.as_ptr().add(n) as *const u8;
+    /// let x = [5u8, 6, 7, 8, 9];
```

Comment:
nit: if you wouldn't mind while you're here, I think we usually prefer
```suggestion
    /// let x = [5_u8, 6, 7, 8, 9];
```